screenshots/configs/core_screenshot_config.yaml

```diff
@@ -574,6 +574,16 @@ quinary:
       await page.waitForDelay(10000);
       page.done();
     message: clicking on "Deaths" tab for AR quinary
+
+  AZ:
+    overseerScript: >
+      page.manualWait();
+      await page.waitForDelay(10000);
+      page.mouse.move(880, 1100);
```

Comment:
thanks @gilmourj ! this one's odd - you need to do both a hover and a click, a click alone won't do it?

Reply:
@julia326 thanks! Good catch, just a click works great.